static/css/pad.css

```diff
@@ -150,7 +150,12 @@ li[data-key=showusers] > a #online_count {
 
 .toolbar ul li a .buttonicon:focus {
     border: 0;
-    outline: none;
```

Comment:
Can we use the same outline style as the one used for the main application (thin dotted, offset -2px)? When I tab through the toolbar without voiceover enable, I don't see an outline at all.